### PR TITLE
Community manager job ad

### PR DESCRIPTION
**Pull request by noamross**
Initial draft post just to test embedding form directly on page. Looks good in local preview.

### CODE REVIEW

**Comment by noamross** [comment]
@maelle I can't seem to get bullets to show up, though I see them in other blog posts (e.g. http://localhost:1313/blog/2021/06/24/news-meta/)

**Comment by karthik** [comment]
LGTM

**Review by karthik — APPROVED** [review]

**Comment by karthik** [comment]
This link
image

pretty much refers to the same page. So maybe skip the link and say apply below?

**Review by maelle — APPROVED** [review]
Maybe we should have a link to the full screen form as well?

**Comment by noamross** [comment]
> This link image
> 
> pretty much refers to the same page. So maybe skip the link and say apply below?

I left the link in so that when the page is scraped/copied, people will always be led to the same page. I thought this better than linking to the standalone form.

File: content/careers/2022-02-02-community-manager.md [hunk 1]
```diff
@@ -3,8 +3,8 @@ slug: "community-manager-2022"
 title: "Hiring our next Community Manager"
 date: 2022-02-02
 expirydate: 2022-03-02
-location: Global, 
-conditions: full-time contract position
+location: Global
+conditions: Full-time contract position
```
Comment (noamross) [inline content/careers/2022-02-02-community-manager.md:7]: These two values get concatenated together as a single phrase, it looks weird without the comma and the second part capitalized. I don't think they are ever used separately.

File: content/careers/2022-02-02-community-manager.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+---
+slug: "community-manager-2022"
+title: "Hiring our next Community Manager"
+date: 2022-02-02
+expirydate: 2022-03-02
```
[the remark this answers is not in the document]
Comment (noamross) [reply content/careers/2022-02-02-community-manager.md:5]: I think so. We want to move quickly.

**Pull request merged**